test/performance/CMakeLists.txt

```diff
@@ -15,8 +15,14 @@ macro (seqan3_benchmark benchmark_cpp)
     seqan3_test_component (target "${benchmark}" TARGET_NAME)
     seqan3_test_component (test_name "${benchmark}" TEST_NAME)
 
+    # force alignment of benchmarked loops so that numbers are reliable:
```

Comment:
Maybe you can add another sentence as to when to disable it?
I remember we discussed that this may not help/ is disruptive when the loops are big?
This could go here, or even just that when benchmarks are weird this could be a option to try to disable?

Reply:
:+1:

Reply:
I addressed this.